be/src/exec/parquet/encoding_plain.h

```diff
@@ -164,6 +191,43 @@ class PlainDecoder<Slice> final : public Decoder {
     size_t _offset = 0;
 };
 
+// plain encoding for boolean type is stored as `Bit Packed`, `LSB` first format
+// more details refer to: https://github.com/apache/parquet-format/blob/master/Encodings.md#PLAIN
+template <>
+class PlainDecoder<bool> final : public Decoder {
```

Comment:
LGTM since "Run Length Encoding / Bit-Packing Hybrid (RLE = 3)" is not widely used now.
so we do not need to consider this encoding here